### PR TITLE
Fix tests, namespace etc.

### PR DESCRIPTION
**Pull request by dawehner**

### CODE REVIEW

File: tests/GraphBuilderTest.php [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+namespace Drupal\Tests\Component\TypeGraph;
+
+use Drupal\Component\TypeGraph\GraphBuilder;
+use Drupal\Core\TypedData\DataDefinition;
+use Drupal\Core\TypedData\ListDataDefinition;
+use Drupal\Core\TypedData\MapDataDefinition;
+use Drupal\Core\TypedData\TypedDataManagerInterface;
+
+/**
+ * @coversDefaultClass \Drupal\TypeGraph\GraphBuilder
```
Comment (technology-amazeelabs) [inline tests/GraphBuilderTest.php]: Misses \Component

**Comment by dawehner** [comment]
Seriously, let's drop scrutinizer as long it doesn't work

**Comment by dawehner** [comment]
Can we merge this? Just scrutinizer doens't really like us. The rest works

**Comment by dawehner** [comment]
<3

**Pull request merged**